public/pages/CreateDetector/containers/CreateDetector.tsx

```diff
@@ -190,6 +199,20 @@ export default class CreateDetector extends Component<CreateDetectorProps, Creat
     });
   }
 
+  async getPlugins() {
+    const { services } = this.props;
+    try {
+      const pluginsResponse = await services.opensearchService.getPlugins();
+      if (pluginsResponse.ok) {
+        this.setState({ plugins: pluginsResponse.response.map((plugin) => plugin.component) });
+      } else {
+        console.error('There was a problem getting plugins list');
```

Comment:
We shouldn't log errors in console. We might have some left in other places but we will clean those up as well.

Reply:
OK, right, I found your newer convention.
I have changed this logs to `errorNotificationToast` helper

Reply:
As discussed, let's use "console.warn" instead for such errors that are not catastrophic failures for the app. The consumers of this plugin can decide to replace these with some analytics api calls of their own if they wish to monitor logs

Reply:
done